post-it/src/addPostIt/AddPostIt.jsx

```diff
@@ -0,0 +1,43 @@
+/* import './EditPostIt.css'; */
+import React from 'react'
+import { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { addPostIt } from '../reducers/postIts';
+
+
+
+
+
+function AddPostIt(){
+
+    const list_post_it=useSelector((state)=>state.postIts.listPostIt);
+
+    const NewPostItPrueba=useSelector((state)=>state.postIts.newPostItPrueba);
+    const dispatch = useDispatch();
+   
+    const [list, setNote]=useState(list_post_it);
+    
+
+    const newNote=()=>{
+ 
+    setNote(list=>[...list,(NewPostItPrueba)]);
```

Comment:
por lo que entiendo estas mandando al reducer la lista completa, que te parece si mejor envías solamente el post-it nuevo y el reducer es el encargado de pushearla dentro de la lista? asi se saca un poco de logica desde el ui(que me llamaron la atencion con eso)